File: iRead/iReader/Main/Main/IRNavigationController.m
//
//  IRNavigationController.m
//  iReader
//
//  Created by zzyong on 2018/7/13.
//  Copyright © 2018年 zouzhiyong. All rights reserved.
//

#import "IRNavigationController.h"

@interface IRNavigationController ()

@end

@implementation IRNavigationController

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    [IRUIUtilites appStatusBarMaxY];
}

- (BOOL)prefersStatusBarHidden
{
    return NO;
}

@end

File: iRead/iRead/Main/Main/IRNavigationController.swift
//
//  IRNavigationController.swift
//  iRead
//
//  Created by zzyong on 2020/2/25.
//  Copyright © 2020 zzyong. All rights reserved.
//

import UIKit

class IRNavigationController: UINavigationController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // 解决 UINavigationController pushViewController 时导航栏黑影问题
        view.backgroundColor = .white
    }
}
